### PR TITLE
[WebDriver BiDi] Fix issue with test using `document.cookie`

### PR DESCRIPTION
**Pull request by Lightning00Blade**
In Chromium you can't set cookies via `document.cookie` in `about:blank` page.
https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/dom/document.cc;l=6225
Navigating to a valid url resolves the issue.

CC @juliandescottes

### CODE REVIEW

File: webdriver/tests/bidi/network/continue_request/headers.py [hunk 1]
```diff
@@ -44,7 +44,14 @@ async def test_override_cookies(
     wait_for_event,
     bidi_session,
     top_context,
+    inline
 ):
+    # Navigate away from about:blank to make sure document.cookies can be used
```
Comment (OrKoN) [inline webdriver/tests/bidi/network/continue_request/headers.py:49]:
```suggestion
    # Navigate away from about:blank to make sure document.cookies can be used.
```

File: webdriver/tests/bidi/network/continue_request/cookies.py [hunk 2]
```diff
@@ -26,7 +26,14 @@ async def test_modify_cookies(
     top_context,
     document_cookies,
     modified_cookies,
+    inline
 ):
+    # Navigate away from about:blank to make sure document.cookies can be used
```
Comment (OrKoN) [inline webdriver/tests/bidi/network/continue_request/cookies.py:31]:
```suggestion
    # Navigate away from about:blank to make sure document.cookies can be used.
```

File: webdriver/tests/bidi/integration/cookies_with_network_events.py [hunk 3]
```diff
@@ -111,10 +111,17 @@ async def test_fetch(
     wait_for_future_safe,
     url,
     domain_1,
+    inline
 ):
     # Clean up cookies in case some other tests failed before cleaning up.
     await bidi_session.storage.delete_cookies()
 
+    # Navigate away from about:blank to make sure document.cookies can be used
```
Comment (Lightning00Blade) [inline webdriver/tests/bidi/integration/cookies_with_network_events.py:119]:
```suggestion
    # Navigate away from about:blank to make sure document.cookies can be used.
```

File: webdriver/tests/bidi/integration/cookies_with_network_events.py [hunk 4]
```diff
@@ -111,10 +111,17 @@ async def test_fetch(
...
 
+    # Navigate away from about:blank to make sure document.cookies can be used.
+    await bidi_session.browsing_context.navigate(
+        context=new_tab["context"], url=inline("<div>foo</div>"),
+        wait="complete"
+    )
+
     cookie_name = "foo"
     cookie_value = "bar"
     # Add `Access-Control-Allow-Origin` header for cross-origin request to work.
```
Comment (OrKoN) [inline webdriver/tests/bidi/integration/cookies_with_network_events.py:127]: is it still remaining a cross-origin request?

Comment (Lightning00Blade) [reply webdriver/tests/bidi/integration/cookies_with_network_events.py:127]: It should have been, but changed to use `url` fixture instead, that has the argument `domain` with default empty string.

**Review by OrKoN — APPROVED** [review]

**Pull request merged**